nn/normalization.py

```diff
@@ -205,7 +205,9 @@ def __init__(self, *,
     self.epsilon = epsilon
     if isinstance(param_shape, nn.Dim):
       param_shape = [param_shape]
-    self.scale = nn.Parameter(shape=param_shape) if scale else None
+    if scale is not None:
+      self.scale = nn.Parameter(shape=param_shape)
+      self.scale.initial = 1.0
```

Comment:
Multiple serious problems.
* `scale` is bool
* You forgot to set `self.scale` in the else case.

```suggestion
    self.scale = None
    if scale:
      self.scale = nn.Parameter(shape=param_shape)
      self.scale.initial = 1.0
```